angularjs-portal-home/src/main/webapp/my-app/layout/partials/home-header.html

```diff
@@ -3,8 +3,8 @@
     <md-switch class="md-accent md-hue-2 app-header-toggle" ng-model="expandedMode"
                aria-label="Toggle expanded widget layout" ng-change="toggleMode(expandedMode)"
                style="margin-top:0;margin-bottom:0;">
-      <span hide-sm hide-xs>Show expanded mode</span>
-      <span hide-gt-sm>Show Expanded</span>
+      <span hide-sm hide-xs>Expand widgets</span>
+      <span hide-gt-sm>Expand Widgets</span>
```

Comment:
Inconsistent capitalization? [Prefer sentence-case](https://richardcooke.info/prefer-sentence-case-to-title-case/)?